### ISSUE TITLE
Test dataset & training recipe

### ISSUE THREAD

**Issue by jy-yuan** [heymesut/SJTU_microe#3]
Dear authors,

Congratulations on your good ranking in the DAC challenge. Thanks for sharing this well-organized codebase.
I'm trying to follow the model training part. Just have two questions here, 1) which test dataset did you use? And 2) which training recipe (options) received good results? (i.e. 70.3% ACC)

Thanks.

**Comment by heymesut** [comment]
You can download the dataset from this link https://byu.app.box.com/s/hdgztcu12j7fij397jmd68h4og6ln1jw. Training options are shown in README.

**Comment by jy-yuan** [comment]
Hi, I would like to know the test set you used, did you randomly split the training set?

**Comment by heymesut** [comment]
Yes, we sampled 1000 images randomly from the training set as local test set, and DAC SDC evacuated the model on a hidden dataset
